### DISCUSSION TITLE
Ability to pre-render the docs?

### DISCUSSION THREAD

**Discussion by mattoni** [scalar/scalar#1438] [General]
Hi,

I have a rather beefy spec file I want to render documentation for. As it stands, it seems the docs are being rendered per request, and each request is taking > 5s to render the docs, where I'm left with a pure white page.
![image](https://github.com/scalar/scalar/assets/5110855/388fbc06-a36e-483a-bc55-0973c61063ce)

Is there a way to have the docs be "pre rendered" so that I can make it an atomic release? Otherwise, this may be unusable for larger specs.

**Comment by amritk** [comment]
Yes this will be coming soon!
